scripts/data/supersearch_fields.json

```diff
@@ -3450,7 +3386,7 @@
         "query_type": "enum",
         "permissions_needed": [],
         "storage_mapping": {
-            "type": "string"
+            "type": "integer"
```

Comment:
This switched from `string` to `integer`. Does that sound right? Are all arm version values strictly integers?